core/src/main/java/hivemall/evaluation/AUCUDAF.java

```diff
@@ -110,7 +110,7 @@ public ClassificationEvaluator() {}
 
         @Override
         public ObjectInspector init(Mode mode, ObjectInspector[] parameters) throws HiveException {
-            assert (parameters.length == 2 || parameters.length == 3) : parameters.length;
+            assert (0 < parameters.length && parameters.length <= 3) : parameters.length;
```

Comment:
In Spark, this assertion fails because Spark passes a single parameter in `parameters` here for final output (IIUC [`AUC` finally outputs a single double-typed value for each group](https://github.com/apache/incubator-hivemall/pull/122/files#diff-9d758588c8fad559a15d0b2362e757b2R1134)). In Hive, does this work well?

Reply:
It would be a bug and should use Preconditions. I guess enable assertion `-ea`  is not enabled (cc: @takuti )